tests/Integration/Behaviour/Features/Context/AbstractPrestaShopFeatureContext.php

```diff
@@ -41,11 +41,19 @@ abstract class AbstractPrestaShopFeatureContext implements BehatContext
 
     protected function checkFixtureExists(array $fixtures, $fixtureName, $fixtureIndex)
     {
+        $searchLength = 10;
+
         if (!isset($fixtures[$fixtureIndex])) {
             $fixtureNames = array_keys($fixtures);
-            $firstFixtureNames = array_splice($fixtureNames, 0, 5);
+            $firstFixtureNames = array_splice($fixtureNames, 0, $searchLength);
             $firstFixtureNamesStr = implode(',', $firstFixtureNames);
-            throw new \Exception($fixtureName . ' named "' . $fixtureIndex . '" was not added in fixtures. First 5 added are: ' . $firstFixtureNamesStr);
+            throw new \Exception(sprintf(
```

Comment:
Improved a bit this helper/debug function

Reply:
Do you think this could be removed by `RuntimeException` to homogenize with all the other context classes? (with use statement to avoid the \)

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/18321/commits/6dd5baa266fbae61f58215040303d7d664442769